src/utils/api-requests.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios"
+
+const apiClient  = axios.create({
+    baseURL: "https://charlies-nc-news.onrender.com/api",
+    timeout: 2000,
+})
+
+export function getAllArticles(){
+    return apiClient.get('/articles?limit=500')
```

Comment:
It doesn't matter much here but axios comes with a params object that when used will make any extension much easier